test/MOIWrapper.jl

```diff
@@ -15,6 +15,13 @@ const config = MOIT.TestConfig(atol=1e-4, rtol=1e-4)
 end
 
 @testset "Continuous conic problems" begin
+    exclude = ["sdp", "rootdet", "logdet"]
+    @static if !Compat.Sys.iswindows()
+        # Test exp3 fails for unknown reason on Windows 32 and 64 bits,
```

Comment:
The reason is known. It's because the windows binaries are out of date. It will be fixed when they're updated.